libs/common/src/constants/facility-type.ts

```diff
@@ -8,4 +8,15 @@ export const BSS_EWCS_FACILITY_TYPE = {
   LOAN: 'Loan',
 } as const;
 
+export const BOND_TYPE = {
+  ADVANCE_PAYMENT_GUARANTEE: 'Advance payment guarantee',
+  BID_BOND: 'Bid bond',
+  MAINTENANCE_BOND: 'Maintenance bond',
+  PERFORMANCE_BOND: 'Performance bond',
+  PROGRESS_PAYMENT_BOND: 'Progress payment bond',
+  RETENTION_BOND: 'Retention bond',
+  STANDBY_LETTER_OF_CREDIT: 'Standby letter of credit',
+  WARRANTY_LETTER: 'Warranty letter',
```

Comment:
thanks very much for updating this -- Does this newly renamed constant belong here (if it doesnt, because we're using the index file, it should be a painless switch)?

Reply:
Great shout, i've moved `BOND_TYPE` into it's own file